code/datums/outputs.dm

```diff
@@ -0,0 +1,114 @@
+GLOBAL_LIST_EMPTY(outputs_list)
+
+//NOTE: When adding new sounds here, check to make sure they haven't been added already via CTRL + F.
+
+/datum/outputs
+	var/text = ""
+	var/list/sounds = list('sound/items/airhorn.ogg'=1) //weighted, put multiple for random selection between sounds
+	var/list/icon = list('icons/sound_icon.dmi',"circle", HUD_LAYER) //syntax: icon, icon_state, layer
+
+/datum/outputs/New()
+	GLOB.outputs_list[type] = src
+
+/datum/outputs/proc/send_info(mob/receiver, turf/turf_source, vol as num, vary, frequency, falloff, channel = 0, pressure_affected = TRUE)
+	var/sound/S
+	if(receiver.client)
```

Comment:
why do you check for client here?
let the mob proc check self for being possessed by the client